test/unit/spec/common/jobManager-Spec.js

```diff
@@ -1331,60 +1332,73 @@ define([
-                    expect(this.jobManagerInstance.listeners).toEqual({});
+                    const expected = [
+                        [actionRequest, { jobIdList: [JobsData.batchParentJob.job_id] }],
+                    ];
+                    const callArgs = this.bus.emit.calls.allArgs();
+                    expect(callArgs).toEqual(jasmine.arrayWithExactContents(expected));
+                    // the job model should be unchanged
+                    expect(
+                        Object.keys(this.jobManagerInstance.model.getItem('exec.jobs.byId'))
+                    ).toEqual(jasmine.arrayWithExactContents(batchIds));
+                    // there should be a status listener for the batch parent
+                    expect(
+                        this.jobManagerInstance.listeners[JobsData.batchParentJob.job_id]
+                    ).toEqual({ 'job-status': this.bus.listen() });
                 });
             });
 
             describe('resetJobs', () => {
-                it('can reset the stored jobs and listeners', function () {
-                    spyOn(this.bus, 'emit');
+                function runResetTest(ctx, thisCellId = null) {
+                    spyOn(ctx.bus, 'emit');
                     expect(
-                        Object.keys(this.jobManagerInstance.model.getItem('exec.jobs.byId'))
+                        Object.keys(ctx.jobManagerInstance.model.getItem('exec.jobs.byId'))
                     ).toEqual(
                         jasmine.arrayWithExactContents(
                             JobsData.allJobsWithBatchParent.map((jobState) => {
                                 return jobState.job_id;
                             })
                         )
                     );
-                    expect(this.jobManagerInstance.listeners).toBeDefined();
-                    this.jobManagerInstance.resetJobs();
-                    expect(this.jobManagerInstance.model.getItem('exec')).not.toBeDefined();
-                    expect(this.jobManagerInstance.listeners).toEqual({});
+                    expect(ctx.jobManagerInstance.listeners).toBeDefined();
+                    ctx.jobManagerInstance.resetJobs();
+                    expect(ctx.jobManagerInstance.model.getItem('exec')).not.toBeDefined();
+                    expect(ctx.jobManagerInstance.listeners).toEqual({});
                     // check the args to bus.emit were correct
-                    expect(this.bus.emit).toHaveBeenCalled();
-                    const callArgs = this.bus.emit.calls.allArgs();
-                    expect(callArgs).toEqual(jasmine.arrayWithExactContents(resetJobsCallArgs));
+                    expect(ctx.bus.emit).toHaveBeenCalled();
+                    let expected = [resetJobsCallArgs];
+                    if (thisCellId) {
+                        expected = [resetJobsCallArgs, ['reset-cell', { cellId }]];
+                    }
```

Comment:
if there's a `cellId` set in the jobManager (as will happen for any bulk cells), resetting jobs will also include a `reset-cell` message.